buildSrc/src/main/java/Compose.kt

```diff
@@ -1,14 +1,12 @@
 object Compose {
     private const val compostActivity = "1.6.0-alpha05"
-    private const val compostNavigation = "2.5.0-alpha01"
-    private const val composeVersion = "1.3.0-alpha01"
+    private const val composeVersion = "1.2.0-rc03"
```

Comment:
I actually just today released a new version for compose 1.3.
So if you want, I believe you can keep this compose version and use 1.7.15-beta compose destinations.